### PR TITLE
[WIP] Reduce overall size and tightening of .toast-pf alerts on overview

### LINKED ISSUE #238: Close button for builds different size than alerts on overview
This is obviously a minor issue, but I noticed it immediately.

openshift_web_console

@jwforres @rhamilto

### PR DESCRIPTION
**Pull request by sg00dwin**
**Height 36px (updated pr)**
screen shot 2016-11-15 at 11 37 42 am

**Height 31px (initial pr)**
screen shot 2016-11-15 at 11 43 44 am

**Height 45px (current)**
screen shot 2016-11-15 at 11 32 24 am

Fixes #238

### CODE REVIEW

**Comment by spadgett** [comment]
Feels a bit too cramped to me :/

/cc @ajacobs21e

File: app/styles/_toast.less [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//
+// Toast alerts
+// -------------------------------------------------
+
+.alert.toast-pf {
```
Comment (spadgett) [inline app/styles/_toast.less]: I think we'd only want to change overview alerts, not all toast notifications, which we might start using someday.

**Comment by sg00dwin** [comment]
@jwforres @spadgett 
- single line height reduced to 31px. 
- size consistent with pipeline-collapsed state, close 'x' size and positioning

**Comment by sg00dwin** [comment]
Now slightly taller
updated pr and screenshots

**Comment by jwforres** [comment]
I like the 36px version, seems like a happy medium

**Comment by ajacobs21e** [comment]
@sg00dwin yep 36 looks good!

**Comment by sg00dwin** [comment]
Rebased PR

**Comment by rhamilto** [comment]
@sg00dwin, did you also reduce the size of the close (X) icon?  See https://github.com/openshift/origin-web-console/issues/238

**Comment by sg00dwin** [comment]
@rhamilto I did, they are now both `font-size:16px` 
https://github.com/openshift/origin-web-console/blob/master/app/styles/_pipeline.less#L541

**Comment by rhamilto** [comment]
@sg00dwin.  Awesome.  That makes this a two-fer!  Make sure you close #238 when this merges.

**Comment by sg00dwin** [comment]
@jwforres rebased pr

**Comment by jwforres** [comment]
[merge]

**Pull request merged**